lib/matplotlib/axes/_base.py

```diff
@@ -17,6 +17,7 @@
 from matplotlib import cbook
 from matplotlib.cbook import (_check_1d, _string_to_bool, iterable,
                               index_of, get_label)
+from collections import OrderedDict
```

Comment:
To be clear, it should be somewhere in the block at lines 7-10 (for standard library imports).